react/Pill/Pill.test.js

```diff
@@ -15,6 +15,11 @@ describe('Pill:', () => {
     expect(pill).toMatchSnapshot();
   });
 
+  it('should render a child component', () => {
+    const ChildComponent = () => (<div>Hello</div>);
+    expect(render(<Pill text={<ChildComponent />} />)).toMatchSnapshot();
```

Comment:
You can use shallow here as well, because in this case we do not care what are actual html elements of a child

Reply:
Can you use the exisiting `renderPill` function to keep this test consistent with the others in this file?